movie-app1.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
The app is half functional as it displays movie info albeit it doesn't remove info from the previous selection. Comment section also requires some work but I think you got this if you spend more time on it. Here are some notable feedbacks: 
1. CSS requires more work as you didn't follow many of guideline indicated on the rubric (i.e. app size, missing image in the header, missing h3 tag for the movie info). 
2. New movie info to replace old movie info when user chooses new movie. 
3. Comment section needs to be completed. 
4. Some issue with selecting DOM element because they are outside of "DOMContentLoaded" event listener. 

Please review, revise and revisit this assessment and try to finish all its missing/malfunctioning features since you will work with similar functionalities a lot in the future. Also reach out to me or other instructional team members at any time if you need help.